apps/lobe-chat/1.36.18/docker-compose.yml

```diff
@@ -1,6 +1,6 @@
 services:
   lobe-chat:
-    image: lobehub/lobe-chat:v1.36.18
+    image: lobehub/lobe-chat:v1.36.19
     container_name: ${CONTAINER_NAME}
     restart: always
     networks:
```

Comment:
No irregularities were found in the provided code snippet. However, you can consider updating the Docker image to v1.36.21 instead of v1.36.19 if there is an official update available after September 1st, 2021. Additionally, ensure that `${CONTAINER_NAME}` is properly defined and does not contain special characters. The current configuration appears standard for managing a service with continuous restarts on default networks.